src/app/server/Server.cpp

```diff
@@ -150,9 +150,9 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     err = mTransports.Init(UdpListenParameters(DeviceLayer::UDPEndPointManager())
                                .SetAddressType(IPAddressType::kIPv6)
                                .SetListenPort(mSecuredServicePort)
-#if CHIP_SYSTEM_CONFIG_USE_OPEN_THREAD_UDP
+#if CHIP_SYSTEM_CONFIG_USE_OPEN_THREAD
```

Comment:
Can it be called `CHIP_SYSTEM_CONFIG_USE_OPEN_THREAD_ENDPOINT`? Not all OpenThread platforms need a dedicated endpoint, so this can be misleading.

Reply:
Sure, will rename

Reply:
Thanks!